tests/model/schema/schema.js

```diff
@@ -641,4 +641,40 @@ describe( 'Schema', () => {
 			expect( result[ 1 ].end.path ).to.members( [ 1 ] );
 		} );
 	} );
+
+	describe( 'getLimitElement()', () => {
```

Comment:
Ha... there's also a case with selection spanning over multiple roots, but let's assume that this is invalid for now (cause it'd break lots of other things too).